src/nn/Compilation.js

```diff
@@ -55,7 +55,8 @@ export default class Compilation {
   async finish() {
     switch (this._backend) {
       case 'WASM': {
-        this._preparedModel = await this._device.prepareModel(this._model);
+        this._preparedModel = new WebGLModel(this._model);
```

Comment:
perhaps we need to change name of `WebGLModel` to `TfjsModel`?

Reply:
Fine, I also want to change its name but don't know what name is suitable.